acceptance/steps/register.go

```diff
@@ -41,4 +41,11 @@ func init() {
 	framework.RegisterStep(`^"([^"]*)" should exist and be able to authenticate to the "([^"]*)" cluster$`, shouldExistAndBeAbleToAuthenticateToTheCluster)
 	framework.RegisterStep(`^"([^"]*)" should be able to authenticate to the "([^"]*)" cluster with password "([^"]*)" and mechanism "([^"]*)"$`, shouldBeAbleToAuthenticateToTheClusterWithPasswordAndMechanism)
 	framework.RegisterStep(`^there should be ACLs in the cluster "([^"]*)" for user "([^"]*)"$`, thereShouldBeACLsInTheClusterForUser)
+
+	// Operator scenario steps
+	framework.RegisterStep(`^operator is running$`, operatorIsRunning)
```

Comment:
nit: You'd want a `the` or `a` here: Given _the_ operator is running.

Reply:
Added `the`